packages/twilio-run/src/utils/requireFromProject.ts

```diff
@@ -0,0 +1,6 @@
+import { createRequire } from 'module';
+import { join } from 'path';
+
+export function requireFromProject(projectDir: string, packageName: string) {
+  return createRequire(join(projectDir, 'node_modules'))(packageName);
```

Comment:
This will still fallback to the version installed by `twilio-run` if there isn't an explicit dependency on `twilio` in the project, right? And that is the intention?

Reply:
I made the fallback more explicit now.

Reply:
Have you pushed? The last commits I can see are from 13 days ago.

Reply:
Sorry I was changing around logic afterwards. It's now up-to-date